pylot/perception/detection/detection_operator.py

```diff
@@ -217,3 +232,38 @@ def __run_model(self, image_np):
         res_boxes = boxes[0][:num_detections]
         res_scores = scores[0][:num_detections]
         return num_detections, res_boxes, res_scores, res_classes
+
+    def __run_yolo_model(self, image_np):
```

Comment:
New inference function specific to YOLO. Some notable differences include:
* YOLO requires inputs to be RGB format, but CARLA outputs frames in BGR format.
* YOLO input tensor shape differs from the models in the Tensorflow Object Detection zoo and the Pylot default object detection models.
* YOLO output bounding boxes are not normalized to the range [0, 1] which Pylot expects.